packages/service/src/scripts/run-webpack.ts

```diff
@@ -1,12 +1,11 @@
 import * as webpack from 'webpack';
```

Comment:
This file can be moved to utils package and instead of configName we can feed configPath as argument. But that would make sense if we will have a scope where other packages want to execute webpack commands

Reply:
That's a good one. I will think about it - a generic webpack runner script could be helpful in the future 👍

Reply:
@MohitTilwani15 I added it now as a script in the webpack package so everyone can use it with specifying a path